lib/editing/editor_codemirror.dart

```diff
@@ -122,7 +127,9 @@ class CodeMirrorFactory extends EditorFactory {
       String stringToReplace = doc.getValue().substring(
           result.replaceOffset, result.replaceOffset + result.replaceLength);
 
-      if (hints.isEmpty && ed.completionActive) {
+      if (hints.isEmpty
```

Comment:
Perhaps add a comment - `// Only show 'no suggestions' if the completion was explicitly invoked.